db/bindings.h

```diff
@@ -0,0 +1,118 @@
+void db_bind_node_id(struct db_stmt *stmt, int pos, const struct node_id *ni);
+void db_bind_node_id_arr(struct db_stmt *stmt, int col,
+			 const struct node_id *ids);
+void db_bind_pubkey(struct db_stmt *stmt, int pos, const struct pubkey *p);
+void db_bind_short_channel_id(struct db_stmt *stmt, int col,
+			      const struct short_channel_id *id);
+void db_bind_short_channel_id_arr(struct db_stmt *stmt, int col,
+				  const struct short_channel_id *id);
+void db_bind_signature(struct db_stmt *stmt, int col,
+		       const secp256k1_ecdsa_signature *sig);
+void db_bind_timeabs(struct db_stmt *stmt, int col, struct timeabs t);
+void db_bind_tx(struct db_stmt *stmt, int col, const struct wally_tx *tx);
+void db_bind_psbt(struct db_stmt *stmt, int col, const struct wally_psbt *psbt);
+void db_bind_amount_msat(struct db_stmt *stmt, int pos,
+			 const struct amount_msat *msat);
+void db_bind_amount_sat(struct db_stmt *stmt, int pos,
+			const struct amount_sat *sat);
+void db_bind_json_escape(struct db_stmt *stmt, int pos,
+			 const struct json_escape *esc);
+void db_bind_onionreply(struct db_stmt *stmt, int col,
+			const struct onionreply *r);
+void db_bind_talarr(struct db_stmt *stmt, int col, const u8 *arr);
+
+/* Modern variants: get columns by name from SELECT */
+/* Bridge function to get column number from SELECT
+   (must exist) */
+size_t db_query_colnum(const struct db_stmt *stmt, const char *colname);
+
+u64 db_col_u64(struct db_stmt *stmt, const char *colname);
+size_t db_col_bytes(struct db_stmt *stmt, const char *colname);
+const void* db_col_blob(struct db_stmt *stmt, const char *colname);
+char *db_col_strdup(const tal_t *ctx,
+		    struct db_stmt *stmt,
+		    const char *colname);
+void db_col_preimage(struct db_stmt *stmt, const char *colname, struct preimage *preimage);
+void db_col_amount_msat(struct db_stmt *stmt, const char *colname, struct amount_msat *msat);
+void db_col_amount_sat(struct db_stmt *stmt, const char *colname, struct amount_sat *sat);
+struct json_escape *db_col_json_escape(const tal_t *ctx, struct db_stmt *stmt, const char *colname);
+void db_col_sha256(struct db_stmt *stmt, const char *colname, struct sha256 *sha);
+void db_col_sha256d(struct db_stmt *stmt, const char *colname, struct sha256_double *shad);
+void db_col_secret(struct db_stmt *stmt, const char *colname, struct secret *s);
+struct secret *db_col_secret_arr(const tal_t *ctx, struct db_stmt *stmt,
+				 const char *colname);
+void db_col_txid(struct db_stmt *stmt, const char *colname, struct bitcoin_txid *t);
+void db_col_channel_id(struct db_stmt *stmt, const char *colname, struct channel_id *dest);
+void db_col_node_id(struct db_stmt *stmt, const char *colname, struct node_id *ni);
+struct node_id *db_col_node_id_arr(const tal_t *ctx, struct db_stmt *stmt,
+				   const char *colname);
+void db_col_pubkey(struct db_stmt *stmt, const char *colname,
+		   struct pubkey *p);
```

Comment:
```suggestion
void db_col_pubkey(struct db_stmt *stmt, const char *colname,
		       struct pubkey *p);
```

With Github UI is a little bit difficult, but I think here it is missing a tab